week2/index.js

```diff
@@ -1,65 +1,129 @@
+//Include File File System package.
 var fs = require('fs');
 
+//Get input commands, remove 'node index.js' form the command.
 var options = process.argv.slice(2);
 
-var command = options[0];
-
-switch (command) {
-  case 'help':
-  default:
-    showHelp();
-    break;
-  case 'list':
-    listTodos();
-    break;
+// find the value of the command, call help file in case command value not found.
+switch (options[0]) {
```

Comment:
Just a style tip - when you assign the value of `options[0]` to a meaningful variable name like `command` it helps with clarity. 

So does your comment above, but in general in production code I like to avoid hard-coded things like `options[0]`. Again - just a question of style, not correctness.